source/melpomene/src/sim_drivers/emb_display.rs

```diff
@@ -89,22 +89,27 @@ impl SimDisplay {
 struct CommanderTask {
     kernel: &'static Kernel,
     cmd: KConsumer<Message<EmbDisplayService>>,
-    display_info: DisplayInfo,
+    width: u32,
+    height: u32,
 }
 
 struct Context {
     sdisp: SimulatorDisplay<Gray8>,
+    framebuf: HeapArray<u8>,
     window: Window,
     dirty: bool,
 }
 
 impl CommanderTask {
     /// The entrypoint for the driver execution
-    async fn run(mut self, width: u32, height: u32) {
+    async fn run(self, width: u32, height: u32) {
         let output_settings = OutputSettingsBuilder::new()
             .theme(BinaryColorTheme::OledBlue)
+            .scale(1)
```

Comment:
I should remove this, or we should make it configurable. With hidpi on my mac, the default scale of 2 is huge, where it wasn't quite as huge on linux with no hidpi scaling.